src/app/core/property.model.ts

```diff
@@ -22,7 +22,7 @@ export class Property {
   }
 
   get assessedValue(): string {
-    return `$${this._assessment.total_asmt}`;
+    return `${this._assessment.assessed_value}`;
```

Comment:
Does removing one of the dollar symbols `$` still display the dollar symbol on the search component?

Reply:
The first dollar symbol was being read as a string dollar symbol which is where the $ in $undefined was coming from.